Objects/typeobject.c

```diff
@@ -7684,6 +7685,53 @@ wrap_descr_delete(PyObject *self, PyObject *args, void *wrapped)
     Py_RETURN_NONE;
 }
 
+static PyObject *
+wrap_buffer(PyObject *self, PyObject *args, void *wrapped)
+{
+    PyObject *arg = NULL;
+
+    if (!PyArg_UnpackTuple(args, "", 1, 1, &arg)) {
+        return NULL;
+    }
+    Py_ssize_t flags = PyNumber_AsSsize_t(arg, PyExc_OverflowError);
+    if (flags == -1 && PyErr_Occurred()) {
+        return NULL;
+    }
+
+    return PyMemoryView_FromObjectAndFlags(self, flags);
```

Comment:
You need to convert `Py_ssize_t` to `int` here. Probably with:

`Py_SAFE_DOWNCAST(flags, Py_ssize_t, int))`

Reply:
Thanks, done